staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/marshal.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Is this intentional?  Usually copyright dates are kept at the original (oldest possible) value.

Reply:
Unintentional. Fixed.